CONTRIBUTING.md

```diff
@@ -1,47 +1,104 @@
-```
-    pretalx-postgresql = {
-    imports = [
-      ./pretalx/pretalx.nix
-      ./pretalx/postgresql.nix
-    ];
-```
-This service can then be deployed on NixOS to a local VM running integration tests using the following comamands:
-```sh
-nix build -L .#nixosTests.x86_64-linux.pretalx.driverInteractive
-./result/bin/nixos-test-driver # Start a shell
-```
-
-Once in the spawned shell, you can start a VM that will execute the tests using the following command:
-```python
-start_all() # Run the VM
-```
-More details on running pretalx in a test VM are available in the [README](https://github.com/ngi-nix/NGIpkgs/edit/main/pkgs/pretalx/README.md) for this service.
+# Contributing to NGIpkgs
+
+This document is for people wanting to contribute to the implementation of NGIpkgs. This involves interacting with implementation changes that are proposed using [GitHub](https://github.com/) [pull requests](https://docs.github.com/pull-requests) to the [NGIpkgs](https://github.com/ngi-nix/ngipkgs/) repository (which you're in right now).
+
+This document assumes that you already know:
+
+- How to [create a Nix package](https://nix.dev/tutorials/packaging-existing-software#).
+- How to [use GitHub and Git](https://docs.github.com/en/get-started/quickstart/hello-world).
+
+In addition, a GitHub account is required, which you can sign up for [here](https://github.com/signup).
+
+## How to create pull requests
+
+Packagers are encouraged to contribute NGI projects to Nixpkgs, instead of to this repository. However, there are many reasons for not being able to do so, including:
+
+- Expediting the public availability of a package prior to its acceptance into Nixpkgs and landing in a channel.
+- Package is not a [good candidate for Nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package).
+- Package is a good candidate for Nixpkgs, but no one is willing to be a maintainer :cry:.
+
+In any case, it is encouraged to create a pull request to Nixpkgs, then to this repository, with a comment linking to the pull request to Nixpkgs in the description and the Nix expressions.
+
+---
+
+Now that this is out of the way. To create pull requests to NGIpkgs:
+
+1. Set up a local version of NGIpkgs. If you don't have write access to NGIpkgs, create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#forking-a-repository) of the repository and clone it.
```

Comment:
I'd suggest to directly link to [github.com/ngi-nix/ngipkgs/fork](https://github.com/ngi-nix/ngipkgs/fork), but of course there's value in *additionally* linking to the docs.